janusgraph-dist/src/assembly/static/bin/janusgraph.sh

```diff
@@ -48,7 +48,7 @@ fi
 : ${GSRV_PORT:=8182}
 
 : ${SLEEP_INTERVAL_S:=2}
-VERBOSE=
+VERBOSE=true
```

Comment:
Is this a leftover from debugging or do we really want to enable verbose logging by default?